core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -152,11 +152,13 @@ public KafkaSchemaRegistry(SchemaRegistryConfig config,
     reporters.add(new JmxReporter(jmxPrefix));
     this.metrics = new Metrics(metricConfig, reporters, new SystemTime());
     this.masterNodeSensor = metrics.sensor("master-slave-role");
+
+    Map<String, String> configuredTags = config.getMap(RestConfig.METRICS_TAGS_CONFIG);
     MetricName
         m = new MetricName("master-slave-role", "master-slave-role",
                            "1.0 indicates the node is the active master in the cluster and is the"
                            + " node where all register schema and config update requests are "
-                           + "served.");
+                           + "served.", configuredTags);
```

Comment:
METRICS_TAGS_CONFIG is a config property with low importance. If the user don't set the value, `configuredTags` would be null? Is that ok?

Reply:
Yes these are not mandatory. Tags are useful when teams would have to start differentiating their clusters/DC in their monitoring system